hoodie-common/src/main/java/com/uber/hoodie/common/util/collection/converter/HoodieRecordConverter.java

```diff
@@ -39,27 +33,12 @@
 public class HoodieRecordConverter<V> implements
```

Comment:
this class is now effectively a static method?  Can we clean up and get rid of the `Converter` hierarchy and just have two util methods?

Reply:
There's a StringConverter class and this hierarchy is to allow for more converters if spillable is to be used elsewhere, let's keep it ?

Reply:
Serialization effectively will subsume the need here right.. i.e String is Serializable anyway.. I don't think this is needed per se, if we have serializable requirements for keys too for the SpillableMap. How much work is it to remove this?